server/protocols/jmap/src/main/java/org/apache/james/jmap/model/MessagePreviewGenerator.java

```diff
@@ -35,6 +35,6 @@ public String compute(Optional<String> textBody) {
     }
 
     private String truncateToMaxLength(String body) {
-        return StringUtils.left(body, MAX_PREVIEW_LENGTH);
+        return StringUtils.left(StringUtils.normalizeSpace(body), MAX_PREVIEW_LENGTH);
```

Comment:
Where is the associated test ? If it's in a previous or next commit I think you should merge those commit